tests/core/test_index.py

```diff
@@ -54,6 +54,16 @@ def test_supplement_index_with_system():
     assert cuda_pkg.package_type == PackageType.VIRTUAL_SYSTEM
 
 
+def test_supplement_index_with_system_glibc():
```

Comment:
This test will fail on non-glibc systems

Reply:
@scopatz, the tests fail because `context.libc_family_version` is not overriden by the env variable `CONDA_OVERRIDE_GLIBC`